### DISCUSSION TITLE
"per-day" versus "per-payout" format

### DISCUSSION THREAD

**Discussion by jbowman5** [w3f/staking-rewards-collector#54] [General]
I personally need the per-day format, as some chains like KILT payout every 6 minutes.  My 'daily' reward sum is going right into Koinly as an entry.   Right now, you can download per-payout CSV directly from subscan blockchain explorer.
What previous version should I run to use the per-day format?

'

**Comment by jonasW3F** [comment]
As far as I can remember, the PR from @rphmeier changed it to a per-day format here https://github.com/w3f/staking-rewards-collector/commit/c98220176bc3f364050b373d0a19f1977d241725 - you can try to use the version before that, but I am not sure whether it already supported KILT.

My advice would be to get the per-payout CSV with the newest version and aggregate the data on a daily basis. Dates are properly formatted, so it should be easy to do. I'd know how to quickly do it with R, but I assume there is also a way in Excel or other programs.
